### PR TITLE
actualize the se inline deployment guide

### PR DESCRIPTION
**Pull request by AnastasiaTWW**

### CODE REVIEW

File: docs/latest/installation/security-edge/deployment.md [hunk 1]
```diff
@@ -15,34 +15,95 @@ Security Edge service provides a secure cloud environment where the Wallarm node
 * Reduced costs: lower operational overhead with Wallarm-managed nodes, allowing faster deployment and scalability.
 * Seamless integration: simple configuration with your existing CDN or load balancers, allowing you to protect your API landscape without disruptions.
 
-## Running the Edge inline node
+Only one Edge inline node deployment is allowed per account. In this single deployment, you can configure multiple origins to forward traffic to and multiple hosts to protect.
```
Comment (xDmitriev) [inline docs/latest/installation/security-edge/deployment.md]: why do we mention that?

File: docs/latest/installation/security-edge/deployment.md [hunk 2]
```diff
@@ -15,34 +15,95 @@ Security Edge service provides a secure cloud environment where the Wallarm node
...
 
-1. The Security Edge deployment is available only with the corresponding subscription. Contact sales@wallarm.com to obtain it.
-1. Go to the Wallarm Console → **Security Edge** → **Edge inline** → **Add origin**.
-1. Specify the **origin** (source server or infrastructure) where the Edge node will forward the filtered traffic. This can be either an IP address or a domain.
-1. Choose one or more **regions** to deploy the Wallarm node.
+## Deploying the Edge inline node
```
Comment (xDmitriev) [inline docs/latest/installation/security-edge/deployment.md]: [Dushnila mode: on]
Probably, configure? The user does not deploy anything. Yes, our platform is deploying Nodes under the hood, as well as issuing certs and etc. But in general user input - is the configuration which is handling by our platform.
[Dushnila mode: off]

File: docs/latest/installation/security-edge/deployment.md [hunk 3]
```diff
@@ -15,34 +15,95 @@ Security Edge service provides a secure cloud environment where the Wallarm node
...
+In **General settings**, specify the following:
+
+1. Choose one or more **regions** to deploy the Edge node.
 
     We recommend selecting regions close to where your APIs or applications are hosted. Deploying in multiple regions enhances geo-redundancy and ensures high availability.
-1. In the **Hosts** section, specify the domains or subdomains that will direct traffic to the Wallarm node.
-    
-    Set the [Wallarm mode](../../admin-en/configure-wallarm-mode.md) for each host and, if needed, associate the host's traffic with a [Wallarm application](../../user-guides/settings/applications.md).
-1. (Optional) For specific **locations** within hosts, you can adjust the following parameters:
+1. Specify **origins** where the Edge node will forward the filtered traffic. This can be an IP address or a domain.
```
Comment (xDmitriev) [inline docs/latest/installation/security-edge/deployment.md]: * And port. For now each origin is a pair of FQDN or IP and PORT.
* domain -> domain name or FQDN

File: docs/latest/installation/security-edge/deployment.md [hunk 4]
```diff
@@ -15,34 +15,95 @@ Security Edge service provides a secure cloud environment where the Wallarm node
...
+Once configuration is complete, Wallarm will provide a CNAME for each DNS zone to verify ownership and complete certificate issuance.
+
+### 3. Hosts
+
+In the **Hosts** section:
+
+1. Specify the domains or subdomains that will direct traffic to the Wallarm node for analysis. Each host entry must match a DNS zone previously defined in the **Certificates** section.
+1. (Optional) Associate the host's traffic with a [Wallarm application](../../user-guides/settings/applications.md) to categorize and manage different API instances or services on the Wallarm platform.
+1. Set the [Wallarm mode](../../admin-en/configure-wallarm-mode.md) for each host.
+1. For each host, select the TLS policy that defines the TLS version and protocol Wallarm will use to securely manage traffic from that host.
```
Comment (xDmitriev) [inline docs/latest/installation/security-edge/deployment.md]: Not implemented yet

File: docs/latest/installation/security-edge/deployment.md [hunk 5]
```diff
@@ -10,39 +10,110 @@ This is an ideal solution for securing APIs when you can redirect traffic from y
...
+
+Once the certificate CNAME record is applied and verified, the Edge deployment status in the Wallarm Console will change to **Deploying**, indicating that the node deployment has started. This process typically takes ~10 minutes.
+
+When the deployment reaches **Active** status, refresh the Edge nodes page to access the CNAME record for traffic routing. Copy this CNAME and update your DNS settings to route traffic to Wallarm.
+
+DNS changes can take up to 24 hours to propagate. Once propagated, Wallarm will proxy all traffic to your origins and mitigate malicious requests.
 
 ## Limitations
 
-* Second-level domains are not supported  (e.g., instead `domain.com` use `www.domain.com`).
```
Comment (xDmitriev) [inline docs/latest/installation/security-edge/deployment.md:42]: We do support minimum 3rd-level domains for now

File: docs/latest/installation/security-edge/deployment.md [hunk 6]
```diff
@@ -15,34 +15,95 @@ Security Edge service provides a secure cloud environment where the Wallarm node
 * Reduced costs: lower operational overhead with Wallarm-managed nodes, allowing faster deployment and scalability.
 * Seamless integration: simple configuration with your existing CDN or load balancers, allowing you to protect your API landscape without disruptions.
```
[the remark this answers is not in the document]
Comment (AnastasiaTWW) [reply docs/latest/installation/security-edge/deployment.md:16]: Does not it contradict with the statement "Currently, the Edge inline node supports only direct, Internet-facing deployment. It cannot operate behind a third-party service, such as a CDN or DDoS protection provider (e.g., Cloudflare, Akamai), that routes traffic."?

[the remark this answers is not in the document]
Comment (xDmitriev) [reply docs/latest/installation/security-edge/deployment.md:16]: No, no. I meant to replace the phrase `load balancer` with `third-party services acting as reverse proxy`

CDN, DDOS protection services such Cloudflare, Akamai - all of them are `third-party services acting as reverse proxy` We do support them

File: docs/latest/installation/security-edge/deployment.md (hunk 4 above)
Comment (xDmitriev) [reply docs/latest/installation/security-edge/deployment.md]:
```
For each host, select the TLS policy that defines the TLS version and protocol Wallarm will use to securely manage traffic from that host.
```
@AnastasiaTWW this is not implemented yet

File: docs/latest/installation/security-edge/deployment.md [hunk 7]
```diff
@@ -1,48 +1,117 @@
...
 
-* Turnkey deployment: deploy Wallarm nodes in globally distributed locations with minimal setup.
-* Autoscaling: node instances automatically scale to handle varying traffic loads.
+* Turnkey deployment: minimal setup is required for Wallarm to automatically deploy nodes across globally distributed locations.
+* Autoscaling: nodes automatically scale horizontally to handle varying traffic loads, with no manual configuration needed.
 * Reduced costs: lower operational overhead with Wallarm-managed nodes, allowing faster deployment and scalability.
 * Seamless integration: simple configuration with your existing CDN or load balancers, allowing you to protect your API landscape without disruptions.
 
-## Running the Edge inline node
+Currently, the Edge inline node supports only direct, Internet-facing deployment. It cannot operate behind a third-party service, such as a CDN or DDoS protection provider (e.g., Cloudflare, Akamai), that routes traffic.
```
Comment (xDmitriev) [inline docs/latest/installation/security-edge/deployment.md:18]: wrong, please double check my comment above

File: docs/latest/installation/security-edge/deployment.md [hunk 8]
```diff
@@ -1,48 +1,117 @@
...
+In **General settings**, specify the following:
+
+1. Choose one or more **regions** to deploy the Edge node.
 
     We recommend selecting regions close to where your APIs or applications are hosted. Deploying in multiple regions enhances geo-redundancy and ensures high availability.
-1. In the **Hosts** section, specify the domains or subdomains that will direct traffic to the Wallarm node.
-    
-    Set the [Wallarm mode](../../admin-en/configure-wallarm-mode.md) for each host and, if needed, associate the host's traffic with a [Wallarm application](../../user-guides/settings/applications.md).
-1. (Optional) For specific **locations** within hosts, you can adjust the following parameters:
+1. Specify **origins** to which the Edge node will forward filtered traffic. You can enter an IP address, domain name or FQDN, with an optional port (defaults to 443 if unspecified).
```
Comment (AnastasiaTWW) [inline docs/latest/installation/security-edge/deployment.md:35]: to be fixed in hosts

File: docs/latest/installation/security-edge/deployment.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (AnastasiaTWW) [reply docs/latest/installation/security-edge/deployment.md:16]: delete CDN and load balancers mentions

**Pull request merged**